README.md

```diff
@@ -1,24 +1,17 @@
 # miniforge
 [![Build Status](https://travis-ci.com/conda-forge/miniforge.svg?branch=master)](https://travis-ci.com/conda-forge/miniforge)
 
-This repository holds a minimal installer for conda on platforms that conda-forge supports but that aren't supported by [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
-
-Relevant conversations:
-
-- https://github.com/conda-forge/conda-forge.github.io/issues/871#issue-496677528
-- https://github.com/conda-forge/conda-forge.github.io/pull/922
+This repository holds a minimal installer for conda specific to conda-forge. It can be directly compared to [Miniconda](https://docs.conda.io/en/latest/miniconda.html), with the added feature that conda-forge is the default channel.
```

Comment:
Is this change needed? It removes the "... that aren't supported by Miniconda" part, which is not (yet?) true.

Reply:
It's a little forward looking considering many people want a conda-forge miniconda. 

Even though, I'm against it, I might merge in the PR that would make x86 builds on linux